src/Application/User/Command/MarkUserActiveCommand.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\User\Command;
+
+use App\Application\CommandInterface;
+use App\Domain\User\User;
+
+final class MarkUserActiveCommand implements CommandInterface
```

Comment:
```suggestion
final class MarkUserAsActiveCommand implements CommandInterface
```

Petite suggestion de nommage